FabLabKasse/gui.py

```diff
@@ -777,15 +788,14 @@ def updateOrder(self, selectLastItem=False):
         total = self.shoppingBackend.get_current_total()
         self.summe.setText(self.shoppingBackend.format_money(self.shoppingBackend.get_current_total()))
 
-
         # disable "pay now" button on empty bill
         self.pushButton_payup.setEnabled(total > 0)
+        self.pushButton_clearCart.setEnabled(total > 0)
```

Comment:
this is wrong: put 0 pieces of an article into the cart. Now "clear cart" is still disabled from this code.
I think setEnabled(True) would be correct.